### PR TITLE
Docs Update

### LINKED ISSUE #1889: Document operational recommendations for worker nodes
Determine the recommended system requirements for each actor.

### LINKED ISSUE #2219: Hardware Wallet Support Documentation

### LINKED ISSUE #2250: Update documentation since Keystore signer can now be used with a Staker operations
Currently in our docs, we indicate that the keystore signer should only be used with Ursula operations, but it can now be used with Staker operations as well.

### PR DESCRIPTION
**Pull request by derekpierre**
- [x] Update demos documentation regarding use of specific ursula fleet (with 12) instead of common fleet (with 5)
- [x] Fix broken links on Installation page
- [x] Document operational requirements for Workers (Fixes #1889)
- [x] Document Keystore use with staker but only for testnet - (Fixes #2250 )

### CODE REVIEW

File: docs/source/guides/ethereum_node.rst [hunk 1]
```diff
@@ -257,7 +257,8 @@ Signing with Local Keystore
 
 .. important::
 
-    Can only be used for :ref:`Worker (Ursula) <ursula-config-guide>` operations.
+    For operational security, the Keystore signer is not recommended for :ref:`Staker operations <staking-guide>`.
+    An exception can be made for testnets, but Staker operations should be performed using a hardware wallet.
```
Comment (KPrasch) [inline docs/source/guides/ethereum_node.rst:261]: :+1:

File: docs/source/guides/installation_guide.rst [hunk 2]
```diff
@@ -135,6 +134,11 @@ Pipenv Installation
 Docker Installation
 -------------------
 
+.. important::
```
Comment (KPrasch) [inline docs/source/guides/installation_guide.rst:137]: Perhaps just a note will do.

**Review by KPrasch — APPROVED** [review]
Approved with comments

File: docs/source/guides/installation_guide.rst (hunk 2 above)
Comment (derekpierre) [reply docs/source/guides/installation_guide.rst]: ✔️

**Review by arjunhassard — APPROVED** [review]
👌 12 Angry Ursulas

**Review by vzotova — APPROVED** [review]

**Pull request merged**